gui/default/syncthing/settings/advancedSettingsModalView.html

```diff
@@ -197,21 +193,21 @@ <h4 class="panel-title" tabindex="0" translate>Default Ignore Patterns</h4>
                   </div>
                 </div>
               </div>
-            </form>
+            </div>
           </div>
         </div>
       </div>
-    </div>
 
-    <div class="modal-footer">
-      <button type="button" class="btn btn-primary btn-sm" ng-click="saveAdvanced()" ng-disabled="guiConfig.$invalid || optionsConfig.$invalid || ldapConfig.$invalid || advancedFolders.$invalid || advancedDevices.$invalid || advancedDefaults.$invalid || (guiConfig.$pristine && optionsConfig.$pristine && ldapConfig.$pristine && advancedFolders.$pristine && advancedDevices.$pristine && advancedDefaults.$pristine)">
-        <span class="fas fa-check"></span>&nbsp;<span translate>Save</span>
-      </button>
-      <button type="button" class="btn btn-default btn-sm" data-dismiss="modal">
-        <span class="fas fa-times"></span>&nbsp;<span translate>Close</span>
-      </button>
-    </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-primary btn-sm" ng-click="saveAdvanced()" ng-disabled="advancedSettingsEditor.$invalid || advancedSettingsEditor.$pristine">
+          <span class="fas fa-check"></span>&nbsp;<span translate>Save</span>
+        </button>
+        <button type="button" class="btn btn-default btn-sm" data-dismiss="modal">
+          <span class="fas fa-times"></span>&nbsp;<span translate>Close</span>
+        </button>
+      </div>
 
+  </form>
```

Comment:
Indent +2
```suggestion
    </form>
```

Reply:
Done, but I've moved the form close tag before footer (following the other similar modals).